build.sbt

```diff
@@ -226,15 +226,18 @@ lazy val `integration-test-local` = project
 
 lazy val docs = project
   .in(file("docs"))
-  .enablePlugins(ParadoxPlugin, NoPublish)
+  .enablePlugins(AkkaParadoxPlugin, NoPublish)
   .disablePlugins(BintrayPlugin)
   .settings(
     name := "Akka Management",
-    paradoxGroups := Map("Language" -> Seq("Scala", "Java")),
-    paradoxTheme := Some(builtinParadoxTheme("generic")),
-    paradox in Compile := (paradox in Compile).dependsOn(LocalRootProject / ScalaUnidoc / doc).value,
-    paradoxProperties ++= Map(
+    publish / skip := true,
+    whitesourceIgnore := true,
```

Comment:
Shouldn't this already be part of `NoPublish` above?

Reply:
I'll drop `NoPublish`, all other modules used `publish / skip` already anyway.